packages/block-library/src/navigation/index.php

```diff
@@ -123,6 +123,16 @@ function block_core_navigation_add_directives_to_submenu( $w, $block_attributes
 				$w->set_attribute( 'data-wp-bind--aria-expanded', 'selectors.core.navigation.isMenuOpen' );
 			};
 
+			// Add directives to the submenu.
+			if ( $w->next_tag(
+				array(
+					'tag_name'   => 'UL',
+					'class_name' => 'wp-block-navigation-submenu',
```

Comment:
Right now, the fix doesn't work with [the Page list block](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/page-list). It seems it doesn't have the `wp-block-navigation-submenu`. I believe we should either check the class `wp-block-navigation__submenu-container` or don't check any class. But I guess the first one should be safer.

Reply:
Good catch. Changed.